### PR TITLE
Add Neptune Summary API support

### PR DESCRIPTION
**Pull request by michaelnchin**
Issue #, if available: N/A

Description of changes:
- Added new `%summary` magic and additional options to `%statistics` to support querying the Neptune [Graph Summary API](https://docs.aws.amazon.com/neptune/latest/userguide/neptune-graph-summary.html). 
- Usage documentation: https://docs.aws.amazon.com/neptune/latest/userguide/notebooks-magics.html#notebooks-line-magics-statistics

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/graph_notebook/magics/graph_magic.py [hunk 1]
```diff
@@ -403,26 +404,67 @@ def statistics(self, line, local_ns: dict = None):
...
         parser.add_argument('--store-to', type=str, default='')
 
         args = parser.parse_args(line.split())
+        mode = args.mode
 
-        if args.mode not in STATISTICS_MODES:
-            print(f'Invalid mode. Please specify one of: {STATISTICS_MODES}, or leave blank to retrieve status.')
-            return
+        if args.summary:
+            if mode not in SUMMARY_MODES:
```
Comment (bechbd) [inline src/graph_notebook/magics/graph_magic.py:418]: Couldn't this check come before the if statement to remove the redundancy

File: src/graph_notebook/neptune/client.py [hunk 2]
```diff
@@ -744,22 +745,30 @@ def _query_status(self, language: str, *, query_id: str = '', **kwargs) -> reque
...
-    def statistics(self, language: str, mode: str = '') -> requests.Response:
+    def statistics(self, language: str, summary: bool = False, mode: str = '') -> requests.Response:
         headers = {
             'Accept': 'application/json'
         }
-        if language in ["pg", "gremlin"]:
-            language = "propertygraph"
-        elif language == "rdf":
-            language = "sparql"
+        if language == "gremlin":
```
Comment (bechbd) [inline src/graph_notebook/neptune/client.py:752]: Do we need a check for OC in here as well?

**Review by bechbd — CHANGES_REQUESTED** [review]

File: src/graph_notebook/magics/graph_magic.py (hunk 1 above)
Comment (michaelnchin) [reply src/graph_notebook/magics/graph_magic.py:418]: Thanks, cleaned up the check.

File: src/graph_notebook/neptune/client.py (hunk 2 above)
Comment (michaelnchin) [reply src/graph_notebook/neptune/client.py:752]: Good idea, thanks. Added `opencypher` and `oc` as valid language options.

**Review by bechbd — APPROVED** [review]

**Pull request merged**